dali/python/nvidia/dali/pipeline.py

```diff
@@ -147,10 +147,12 @@ class Pipeline(object):
         More details can be found in
         `this documentation section <advanced_topics_checkpointing.html>`_.
     py_num_workers : int, optional, default = 1
-        The number of Python workers that will process ``ExternalSource`` callbacks.
-        The pool starts only if there is at least one ExternalSource with ``parallel`` set to True.
+        The number of Python workers that will process :meth:`~nvidia.dali.fn.external_source`
```

Comment:
While at it - I'd mention in this line that it's about _parallel_ external source callbacks.
```suggestion
        The number of Python workers that will process parallel :meth:`~nvidia.dali.fn.external_source`
```